CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ find_package(catkin REQUIRED
     moveit_ros_planning_interface
     pluginlib
     geometric_shapes
+    moveit_visual_tools
```

Comment:
I've considered this. We would also probably want to move rviz_visual_tools in too, since it depends on it. But I've poured many years of work into those two packages though and kind of like to keep it under my name. Otherwise I'd be more restricted in developing it further - its kind of my favorite pet project repo. I've released it into every version of ROS since Hydro and faster than MoveIt! gets released. I'd rather keep it separate in the short term.

btw have you tried it? its a really powerful package IMHO :)